NAMESPACE

```diff
@@ -0,0 +1,5 @@
+# Generated by roxygen2: do not edit by hand
+
+export(EuclideanDT_inC)
```

Comment:
This function no longer exists, so we should remove this export line. There is a compilation warning otherwise.